data_dir/scylla.yaml

```diff
@@ -118,6 +118,20 @@ backends: !mux
             ami_db_cassandra_user: 'ubuntu'
             ami_id_monitor: 'ami-79d3f06e'
             ami_monitor_user: 'centos'
+        #multiple datacenters
+        us_east_1_and_us_west_2:
```

Comment:
Can we have something like?
multi-DC: true
multi-regions: true 
number-of-regions: 2

In this case it will select the first 2 regions in the selected backend_mux for the run.